examples/blender/blender.ts

```diff
@@ -1,7 +1,7 @@
 import path from "path";
 import dayjs from "dayjs";
 import duration from "dayjs/plugin/duration";
-import { Executor, Task, utils, vm, WorkContext } from "yajsapi";
+import { Executor, Task, utils, vm, WorkContext } from "../../yajsapi";
```

Comment:
Please remove it before merging.

Reply:
sure. in my mind, but thanks for reminding!